src/Form/Helper/FormHelper.php

```diff
@@ -11,39 +12,43 @@
 use Symfony\Component\Form\Extension\Core\Type\EmailType;
 use Symfony\Component\Form\Extension\Core\Type\IntegerType;
 use Symfony\Component\Form\Extension\Core\Type\NumberType;
+use Symfony\Component\Form\Extension\Core\Type\RangeType;
 use Symfony\Component\Form\Extension\Core\Type\TextType;
 use Symfony\Component\Form\Extension\Core\Type\TimeType;
 use Symfony\Component\Form\Extension\Core\Type\UrlType;
 
 class FormHelper
 {
     /**
-     * @param object $schema
+     * @param stdClass $property
      * @return string|null
      */
-    public static function resolveFormType($schema): ?string
+    public static function resolveFormType(stdClass $property): ?string
```

Comment:
```suggestion
    public static function resolveFormType(\stdClass $property): ?string
```
and remove `use stdClass;`